### ISSUE TITLE
1 means not_entailment in RTE?

### ISSUE THREAD

**Issue by WyZHANG2018** [pytorch/fairseq#1044]
I test the fine-tuned model of RTE and find that 1 means not_entailment. Is that right?

**Comment by ngoyal2707** [comment]
Hey, so the way fairseq finetuning treats labels is as just like token dictionary. 
You can find the mapping in of `label_index` i.e. `1` to real label string in `RTE-bin/label/dict.txt`

You can find the mapping by following line also:
```
label_fn = lambda label: roberta.task.label_dictionary.string(
    [label + roberta.task.target_dictionary.nspecial]
)
```

Please take a look at the "Inference on GLUE task" section [here](https://github.com/pytorch/fairseq/blob/master/examples/roberta/README.glue.md)
